Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -151,7 +151,15 @@ function Get-DiscoveryServiceUrl($discoUrl)
     if([string]::IsNullOrEmpty($discoUrl)){
         return "https://$env:computername.$($env:userdnsdomain.tolower())/DiscoveryService/v1"
     }else{
-        return $discoUrl
+		  $hasversion = $discoUrl -match "/v\d"
```

Comment:
should we be scrubbing a trailing "/" as well?   I think other places in the installer trap/fix that.

Reply:
there is only an issue when I was doing a little exploratory with having a trailing backslash in the following scenario. http://localhost/DiscoveryService/. This scenario  http://localhost/DiscoveryService/v1/ still works fine and doesn't append another v1.